### ISSUE TITLE
Publish a file containing the checksums for the release assets.

### ISSUE THREAD

**Issue by sdavids** [OpenVPN/easy-rsa#1207]
Please add a `SHA256SUMS`, `checksums.txt`, or something similarly named alongside the archive files for a GitHub release.

https://github.com/OpenVPN/easy-rsa/releases

Basically:

1. build all archive files
2. `sha256sum -b * | sed 's/\*//g' > SHA256SUMS`
3. upload archive files and `SHA256SUMS`

**Comment by TinCanTech** [comment]
All assets come with gpg signatures, for verification.

**Comment by sdavids** [comment]
Where have you published those?

**Comment by TinCanTech** [comment]
https://github.com/OpenVPN/easy-rsa/releases

**Comment by sdavids** [comment]
Let me rephrase it, where is the fingerprint of the signing key published?

**Comment by sdavids** [comment]
I found it 🤦‍♂️

https://github.com/OpenVPN/easy-rsa/tree/master/release-keys

It could be mentioned under 

https://github.com/OpenVPN/easy-rsa/blob/master/README.md#downloads

though.
